src/main/java/frc/robot/subsystems/Deployer.java

```diff
@@ -67,6 +67,9 @@ public void periodic() {
             SmartShuffleboard.put("Deployer", "encoder", getEncoder());
             SmartShuffleboard.put("Deployer", "Fwd Limt", isDeployerFowardLimitSwitchClosed());
             SmartShuffleboard.put("Deployer", "Rev Limit", isDeployerReverseLimitSwitchClosed());
+            SmartShuffleboard.put("Diagnostics","Deployer", "encoder", getEncoder());
```

Comment:
This is not actually the way to create Diagnostics, look at the updated Climber.java in main for how to do diagnostics for booleans, and drivetrain for diagnostics of encoders